Userland/Libraries/LibGfx/TIFFGenerator.py

```diff
@@ -196,29 +196,22 @@ def export_enum_to_cpp(e: Type[EnumWithExportName]) -> str:
 def export_enum_to_string_converter(enums: List[Type[EnumWithExportName]]) -> str:
     stringifier_internals = []
     for e in enums:
-        single_stringifier = fR"""    if constexpr (IsSame<E, {e.export_name()}>) {{
-        switch (value) {{
-            default:
-                return "Invalid value for {e.export_name()}"sv;"""
+        single_stringifier = fR"""inline StringView name_for_enum_tag_value({e.export_name()} value)
```

Comment:
nit: If we have to put an extra qualifier for functions in header files, we might as well use `constexpr` rather than `inline`.

Reply:
Done!